.travis.yml

```diff
@@ -1,18 +1,22 @@
-dist: xenial
-language: python
-python:
-  - 3.6
+language: c
```

Comment:
Why did you removed then `dist`?

I think you need to use `generic`.

Reply:
Let me try just remove that line and see how it goes.